src/test/regress/expected/upgrade_ref2ref_after.out

```diff
@@ -0,0 +1,169 @@
+SET search_path TO upgrade_ref2ref, public;
+BEGIN;
+SET LOCAL citus.multi_shard_modify_mode TO 'sequential';
+\d ref_table_1
```

Comment:
Minor: `\d` may not be consistent across postgres versions. Is there  any view that you can utilize  in `multi_test_helpers`  to get this information?